pandas/core/arrays/period.py

```diff
@@ -67,7 +71,7 @@ class PeriodArray(PeriodMixin, dtl.DatelikeOps):
     """
     Pandas ExtensionArray for storing Period data.
 
-    Users should use :func:`period_range` to create new instances.
+    Users should use :func:`~pandas.period_array` to create new instances.
     Alternatively, :func:`array` can be used to create new instances
```

Comment:
should "array" be changed to the same pattern as for period_array?

Reply:
I assume this is `numpy.array`?

Reply:
im pretty sure this refers to pd.array

Reply:
Changed in 9c91ee7.